app/lib/BaseModelWithAttributes.php

```diff
@@ -1849,10 +1853,12 @@ public function getAttributeHTMLFormBundle($po_request, $ps_form_name, $pm_eleme
 				
 				$label = (sizeof($va_element_set) > 1) ? $va_label['name'] : '';
 				
-				$show_bundle_codes = $po_request->user->getPreference('show_bundle_codes_in_editor');
-				$bundle_code = $this->tableName().'.'.$t_element->get('element_code').'.'.$va_element['element_code'];
+				if($t_element->get('datatype') == 0){ //Only show field level bundle codes inside containers  
```

Comment:
It would be good to assign datatype to a variable outside of the loop and use that in the comparison